en/add_wagtail_to_your_website/wagtail_integration_adding_posts/README.md

```diff
@@ -0,0 +1,130 @@
+# Adding Posts to Wagtail
+
+> Part of this tutorial is based on the Wagtail Tutorial by TorchBox  
+> (https://docs.wagtail.org/en/stable/getting_started/tutorial.html#).
+
+In the last section, we created a new page model which we used to add content to our homepage called `Homepage`, changed 
+the root of our Wagtail site to use this new homepage. We also added the CSS styling we were already using in our 
+project and now the Wagtail has the same look and feel as our website, which is really pretty amazing!
+
+Now, we need to use our posts model to render the posts we already have on the blog using Wagtail. 
+
+Let's get started!
+
+## Adding BlogPage model to Wagtail for posts
+We already have a `Post` model in our `models.py` that inherits from `django.db.models.Model`. We will leave it as it is
+for now and add a new model, a child model of `HomePage` which we will name `BlogPage`. Like the `HomePage` model, our 
+new models also inherits from `wagtails.models.Page` like its parent model `HomePage`.
+
+We first need to add the following import to our `models.py`:
+
+```python
+from wagtail.search import index
+```
+
+Next we need to add the `BlogPage` model to as shown below:
+
+```python
+class BlogPage(Page):
+    date = models.DateField("Post date")
+    intro = models.CharField(max_length=250)
+    text = RichTextField(blank=True)
+
+    search_fields = Page.search_fields + [
+        index.SearchField('intro'),
+        index.SearchField('text'),
+    ]
+
+    content_panels = Page.content_panels + [
+        FieldPanel('date'),
+        FieldPanel('intro'),
+        FieldPanel('text'),
+    ]
+```
+
+Before we move on, we will make some comparisons between our `Post` and `BlogPage` models. 
+- Our `Post` model contains a field named but we can't use the same field name in our `BlogPage` model because our base 
+model `Page` already contains a field by that name and hence this will cause an error. 
+- Our `Post` model also has fields`author` and `published_date` which we omitted in our `BlogPage` model because our 
+base model `Page` has a field `owner`that represents the `author` and has a `-first_published_at` field that represents 
+the `published_date`. 
```

Comment:
Not sure we want to mention this but it’s pretty common for Wagtail page models to also have an author field. `owner` is useful but it’s not always appropriate as it can’t be set in the CMS. Adding an `author` field tends to be a good thing to cover in a Wagtail site tutorial, as it’s usually done with a vanilla Django model disconnected from `User` – always interesting to see how to manage those vanilla models in Wagtail compared to Django.

Reply:
Noted.